db/seeds.rb

```diff
@@ -22,7 +22,7 @@
 # User
 #############################################
 user = User.create!({
```

Comment:
All of this can go, I made a test the first day and forgot to take it out 🤦‍♀️

Reply:
i love it!